src/pages/workspaces/workspace/useDeleteWorkspaceState.test.ts

```diff
@@ -104,22 +111,23 @@ describe('useDeleteWorkspace', () => {
     }
     asMockedFn((mockListAppsV2 as AjaxAppsContract).listAppsV2).mockResolvedValue([{
       appName: 'example',
-      status: 'running'
+      status: 'provisioning'
```

Comment:
Why was this one changed to provisioning? Asking because you seem to test the provisioning case below at https://github.com/DataBiosphere/terra-ui/pull/3800/files#diff-7aba62bb5bd766039e3cbfde8d59add7921bc1de6cc627ce31386f976b0c9a3eR285

Reply:
The original test was getting into the `isDeleteDisabledFromResources == true` state (which is asserted on L144) by setting up a fake WSM controlled resource. Since that call to WSM is going away, I am setting up this app to be in `provisioning` which is equally non-deletable.